peekingduck/nodes/draw/bbox.py

```diff
@@ -36,24 +36,52 @@ class Node(AbstractNode):
 
         |bbox_labels_data|
 
+        |bbox_scores_data|
+
     Outputs:
         |none_output_data|
 
     Configs:
         show_labels (:obj:`bool`): **default = False**. |br|
             If ``True``, shows class label, e.g., "person", above the bounding
             box.
+        show_scores (:obj:`bool`): **default = False**. |br|
+            If ``True``, shows prediction scores, e.g., "0.99", at the bottom
+            left of the bounding box. (If set as ``True``, the previous node must
+            output |bbox_scores_data|, otherwise the pipeline willbreak.)
+        color_choice (:obj:`List[int]`): **default = None**. |br|
+            Define the color of the bounding box, in BGR format.
+            Defined values have to be integers, and :math:`0 \\leq value \\leq 255`
+            (values out of this range will be clipped).
+
     """
 
     def __init__(self, config: Dict[str, Any] = None, **kwargs: Any) -> None:
         super().__init__(config, node_path=__name__, **kwargs)
 
     def run(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
+
+        # if "bbox_scores" not in inputs:
+        #     bbox_scores = []
+        # else:
+        #     bbox_scores = inputs["bbox_scores"]
+        # Optional Parameter
+        # print(self.__dict__)
```

Comment:
Redundant comments and should be removed.

Reply:
Yes, please remove this commented segment if not used.